src/index.html

```diff
@@ -1,12 +1,508 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header page__header">
+      <a href="/" class="header__link">
+        <img 
+          class="header__logo" 
+          src="images/logo.png" 
+          alt="Logo"
+        >
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__element">
+            <a href="#apple" class="nav__link nav__link--is-active">
```

Comment:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/99331793/185622592-eccb58cd-1581-463d-a3a3-8e467f25ac30.png">

```suggestion
            <a href="#apple" class="nav__link nav__link--is-active is-active">
```